src/shared/pages/team/editTeam/__tests__/editTeam.spec.tsx

```diff
@@ -63,6 +78,21 @@ beforeEach(() => {
     addFormErrorSpy.mockReset();
     clearErrorsSpy.mockReset();
     setMessageSpy.mockReset();
+
+};
+
+beforeEach(() => {
+    setDefaults();
+
+    hasRole.mockImplementation((role: string) => {
+        if (role === 'RENAME_TEAM') {
+            return true;
+        } else if (role === 'REASSIGN_TEAM_UNIT') {
+            return true;
+        }
```

Comment:
Since these two both return true, line 88 can replace like:
`if ((role === 'RENAME_TEAM') || (role === 'REASSIGN_TEAM_UNIT')) {`

and remove the else if